propolis/Cargo.toml

```diff
@@ -21,6 +21,11 @@ viona_api = { path = "../viona-api" }
 usdt = "0.1.11"
 tokio = { version = "1", features = ["full"] }
 futures = "0.3"
+crucible = { git = "https://github.com/oxidecomputer/crucible", rev = "adb2662b", optional = true }
+anyhow = "1"
```

Comment:
I'm only including it for crucible Guest return types in _create - it will be removed after https://github.com/oxidecomputer/crucible/issues/85 is addressed.